Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift

```diff
@@ -2,31 +2,35 @@ import Foundation
 
 @objcMembers
 public class SentryReplayOptions: NSObject, SentryRedactOptions {
-    
+
     /**
      * Enum to define the quality of the session replay.
      */
     @objc
     public enum SentryReplayQuality: Int {
+        static let names = [ "low", "medium", "high"]
+
         /**
          * Video Scale: 80%
          * Bit Rate: 20.000
          */
-        case low
-        
+        case low = 0
+
         /**
          * Video Scale: 100%
          * Bit Rate: 40.000
          */
-        case medium
-        
+        case medium = 1
+
         /**
          * Video Scale: 100%
          * Bit Rate: 60.000
          */
-        case high
+        case high = 2
     }
-    
+
+    static let defaultQuality: SentryReplayQuality = .medium
```

Comment:
I dont think we need to expose this.
```suggestion
    private static let defaultQuality: SentryReplayQuality = .medium
```